h2o-py/h2o/model/model_base.py

```diff
@@ -882,19 +882,22 @@ def _plot(self, timestep, metric, server=False):
         if not server: plt.show()
 
 
-    def partial_plot(self, data, cols, destination_key=None, nbins=20, plot=True, plot_stddev = True, figsize=(7, 10), server=False):
+    def partial_plot(self, data, cols, destination_key=None, nbins=20, weight_column_index=-1,
+                     plot=True, plot_stddev = True, figsize=(7, 10), server=False, add_missing_NA=False):
```

Comment:
rename weight_column_index to weight_column and make it a string.
rename add_missing_NA to include_na

Reply:
Done